app/controllers/api_controller/normalizer.rb

```diff
@@ -19,7 +19,7 @@ def normalize_hash(type, obj, opts = {})
       end
 
       attrs.each do |k|
-        value =  normalize_direct(type, k, obj[k])
+        value =  normalize_direct(type, k, obj.kind_of?(ActiveRecord::Base) ? obj.try(k) : obj[k])
```

Comment:
Yeah, we are getting `Hash` sometimes here. For example `actions` like following are going through there as hashes.

```
"actions" : [ 
      { 
        "href" : "http://localhost:3000/api/tenants/1000000000001",
        "method" : "post",
        "name" : "edit"
      },
```